bin/main

```diff
@@ -1,2 +1,49 @@
 #!/usr/bin/env ruby
-puts "Hello World!"
\ No newline at end of file
+puts 'Welcome to tic-tac-toe!'
+
+# Display board
+board = [
+  1, 2, 3,
+  4, 5, 6,
+  7, 8, 9
+]
+
+puts " #{board[0]} | #{board[1]} | #{board[2]} "
+puts '---+---+---'
+puts " #{board[3]} | #{board[4]} | #{board[5]} "
+puts '---+---+---'
+puts " #{board[6]} | #{board[7]} | #{board[8]} "
+puts "\n"
```

Comment:
**Optional**
You can enclose it in a method. Cause every time it is not gonna be the same. something like.
```ruby
def display_board(board)
  puts " #{board[0]} | #{board[1]} | #{board[2]} "
  puts '---+---+---'
  puts " #{board[3]} | #{board[4]} | #{board[5]} "
  puts '---+---+---'
  puts " #{board[6]} | #{board[7]} | #{board[8]} "
  puts "\n"
end
```